TESTS WITH ERRORS 
test03 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "/Users/k2i/academics/automatedgradingapp/app/testing/kb-inference-ec/gradingtests.py", line 73, in test03
    actual = self.KB.kb_explain(fact4)
  File "/Users/k2i/academics/automatedgradingapp/app/testing/kb-inference-ec/student_code.py", line 202, in kb_explain
    result = result + '\n' + self.kb_helper(support, spaces)
  File "/Users/k2i/academics/automatedgradingapp/app/testing/kb-inference-ec/student_code.py", line 137, in kb_helper
    print(fact_or_rule.statement)
AttributeError: 'list' object has no attribute 'statement'

-------------------------------
TESTS FAILED  
-------------------------------
TESTS SKIPPED 
-------------------------------
